libsolutil/CommonData.h

```diff
@@ -57,10 +60,13 @@ template <class U, class... T> std::multiset<T...>& operator+=(std::multiset<T..
 	return _a;
 }
 /// Concatenate the contents of a container onto a multiset, move variant.
-template <class U, class... T> std::multiset<T...>& operator+=(std::multiset<T...>& _a, U&& _b)
+template <class U, class... T,
+	class = typename std::enable_if<!std::is_lvalue_reference<U>::value>::type>
+std::multiset<T...>& operator+=(std::multiset<T...>& _a, U&& _b)
 {
 	for (auto&& x: _b)
 		_a.insert(std::move(x));
+	_b.clear();
```

Comment:
Is this really needed? "If you move from an object, it is left in a consistent but undefined state."

Reply:
it is not really needed, I just thought it may be more save

Reply:
I will remove the clears now, because it is not really needed ;)